src/browser/dummy_screen.js

```diff
@@ -65,7 +65,7 @@ function DummyScreenAdapter(bus)
     }, this);
     bus.register("screen-update-cursor-scanline", function(data)
     {
-        this.update_cursor_scanline(data[0], data[1]);
+        this.update_cursor_scanline(...data);
```

Comment:
I prefer the explicit style of passing arguments as in the old code.

(or, if you prefer, `const [start, end, max] = data;
this.update_cursor_scanline(start, end, max);`)